deblur/test/test_workflow.py

```diff
@@ -0,0 +1,55 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, The Deblur Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from unittest import TestCase, main
+from StringIO import StringIO
```

Comment:
Remove imports, just let the ones that we are using. Check the travis output to see the PEP8/Flake8 errors. We need to have the tests passing always...